dev-util/capstone/patches/capstone-4.0.1.patchset

```diff
@@ -0,0 +1,80 @@
+From 203f796d921948ea60d9941f4a19317b41d66dbe Mon Sep 17 00:00:00 2001
+From: Calvin Hill <calvin@hakobaito.co.uk>
```

Comment:
The original author should be left as it was.

Reply:
@korli I've added the equivalent change of the first patch directly into the recipe. This makes the original patchset obselete.

Reply:
thanks